.betterer.results

```diff
@@ -26,7 +26,7 @@ exports[`too-much-lint`] = {
       [23, 25, 3, "Unexpected any. Specify a different type.", "193409811"],
       [36, 38, 3, "Unexpected any. Specify a different type.", "193409811"]
     ],
-    "src/module/item/base.ts:2799968150": [
+    "src/module/item/base.ts:1027049593": [
```

Comment:
This is getting reverted; exclude file from PR unless you're intentionally updating it.

Reply:
Gone 👍